src/frontend/src/common/components/library/data_subview/components/DownloadModal/style.ts

```diff
@@ -0,0 +1,116 @@
+import styled from "@emotion/styled";
+import IconButton from "@material-ui/core/IconButton";
+import {
+  Button,
+  Checkbox,
+  fontBodyM,
+  fontBodyS,
+  fontBodyXs,
+  fontHeaderXl,
+  getColors,
+  getSpacings,
+  Props,
+} from "czifui";
+
+export const Header = styled.div`
+  ${fontHeaderXl}
+  ${(props: Props) => {
+    const spacings = getSpacings(props);
+    return `
+      margin-bottom: ${spacings?.s}px;
+    `;
+  }}
+`;
+
+export const Content = styled.div`
+  ${fontBodyS}
+  ${(props) => {
+    const colors = getColors(props);
+    return `
+      color: ${colors?.gray[500]};
+    `;
+  }}
+`;
+
+export const Title = styled.span`
+  ${fontBodyM}
+  ${(props) => {
+    const colors = getColors(props);
+    const spacings = getSpacings(props);
+    return `
+      color: ${colors?.gray[500]};
+      margin-bottom: ${spacings?.l}px;
+    `;
+  }}
+`;
+
+export const CheckBoxInfo = styled.div`
+  position: inline-block;
+  float: left;
+  ${fontBodyS}
```

Comment:
I'd personally put `${fontBodyS}` on top before custom rules, since latter CSS rules prevail, I wanna make sure the custom rules will come out on top!

e.g.,

```ts
${fontBodyS}

position: inline-block;
float: left;

${(props) => { ... }}

```